jobs/build/custom/Jenkinsfile

```diff
@@ -266,6 +259,21 @@ node {
                 }
             }
 
+            stage('push images to qe quay') {
+                if (majorVersion == "4") {
+                    config_dir = "${env.WORKSPACE}/qe_quay_config"
+                    buildlib.registry_quay_qe_login(config_dir)
+                    base_command = "${doozerOpts} ${include_exclude} --profile ${repo_type} --registry-config-dir=${config_dir}"
+                    command = "${base_command} images:push --to-defaults"
+                    command += " --filter-by-os='.*'"
+                    try {
+                        buildlib.doozer command
```

Comment:
doozer retries the push a few times right? in that case i guess we don't need a retry here. that might be worth a comment though.